src/pages/vaults/VaultPage.tsx

```diff
@@ -35,7 +40,12 @@ const VaultPage = () => {
   const dispatch = useAppDispatch();
   useDocumentTitle(stringGetter({ key: STRING_KEYS.MEGAVAULT }));
 
+  useEffect(() => {
+    track(AnalyticsEvents.ViewMegaVaultPage());
+  }, []);
```

Comment:
I don't think we need this event as we capture it with `NavigatePage` in `useAnalytics`

Reply:
Good call!